vote/src/vote_state_view/frame_v1_14_11.rs

```diff
@@ -0,0 +1,109 @@
+use {
+    super::{
+        field_frames::{AuthorizedVotersListFrame, ListFrame, PriorVotersFrame, RootSlotFrame},
+        EpochCreditsListFrame, Field, Result, VoteStateViewError, VotesListFrame,
+    },
+    solana_pubkey::Pubkey,
+    solana_vote_interface::state::BlockTimestamp,
+    std::io::BufRead,
+};
+
+#[derive(Debug, Clone)]
+#[cfg_attr(feature = "frozen-abi", derive(AbiExample))]
+pub(super) struct VoteStateFrameV1_14_11 {
+    pub(super) num_votes: u8,
+    pub(super) has_root_slot: bool,
+    pub(super) num_authorized_voters: u8,
+    pub(super) num_epoch_credits: u8,
```

Comment:
why not just store the frames here instead of these values used to construct frames on demand?

Reply:
Saves memory

Reply:
done: https://github.com/anza-xyz/agave/pull/4834/commits/6eff0c14c9bc481bdd568bfc8ab85426d4a3a180